[2022-04-28 15:37:17,922] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: cdn_daily.import_cat.check_presence_of_cat2 manual__2022-04-28T15:37:16.597016+00:00 [queued]>
[2022-04-28 15:37:17,940] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: cdn_daily.import_cat.check_presence_of_cat2 manual__2022-04-28T15:37:16.597016+00:00 [queued]>
[2022-04-28 15:37:17,942] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-28 15:37:17,943] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-04-28 15:37:17,945] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-28 15:37:17,963] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): import_cat.check_presence_of_cat2> on 2022-04-28 15:37:16.597016+00:00
[2022-04-28 15:37:17,970] {standard_task_runner.py:52} INFO - Started process 1741 to run task
[2022-04-28 15:37:17,977] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'cdn_daily', 'import_cat.check_presence_of_cat2', 'manual__2022-04-28T15:37:16.597016+00:00', '--job-id', '1438', '--raw', '--subdir', 'DAGS_FOLDER/cdn_daily.py', '--cfg-path', '/tmp/tmpup6em2_5', '--error-file', '/tmp/tmpqcmdl2es']
[2022-04-28 15:37:17,978] {standard_task_runner.py:77} INFO - Job 1438: Subtask import_cat.check_presence_of_cat2
[2022-04-28 15:37:18,083] {logging_mixin.py:109} INFO - Running <TaskInstance: cdn_daily.import_cat.check_presence_of_cat2 manual__2022-04-28T15:37:16.597016+00:00 [running]> on host d5ad096ffea5
[2022-04-28 15:37:18,215] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cdn_daily
AIRFLOW_CTX_TASK_ID=import_cat.check_presence_of_cat2
AIRFLOW_CTX_EXECUTION_DATE=2022-04-28T15:37:16.597016+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-28T15:37:16.597016+00:00
[2022-04-28 15:37:18,256] {python.py:175} INFO - Done. Returned value was: None
[2022-04-28 15:37:18,330] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=cdn_daily, task_id=import_cat.check_presence_of_cat2, execution_date=20220428T153716, start_date=20220428T153717, end_date=20220428T153718
[2022-04-28 15:37:18,394] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-28 15:37:18,460] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
